php-classes/Slate/CBL/StudentCompetency.php

```diff
@@ -21,10 +21,12 @@ class StudentCompetency extends \ActiveRecord
             'type' => 'uint',
             'index' => true
         ],
-        'CompletionType' => [
-            'type' => 'enum',
-            'values' => array('demonstrated', 'overridden')
-        ]
+        'Status' => [
+            'type' => 'enum'
+            ,'values' => ['enrolled', 'completed']
+            ,'default' => 'enrolled'
+        ],
+        ['Level' => 'uint']
```

Comment:
This field definition shouldn't be wrapped in square brackets, the name of the field is supposed to the key in the top-level $fields array, this array value has no key currently